tests/unit/inference/test_export.py

```diff
@@ -26,11 +29,6 @@
 TRITON_SERVER_PATH = find_executable("tritonserver")
 configure_tensorflow()
```

Comment:
you have to do this before youattempt to load in tensorflow

Reply:
@jperez999 changed